Data/src/main/java/org/tribuo/data/columnar/processors/response/QuartileResponseProcessor.java

```diff
@@ -33,9 +39,11 @@
 public class QuartileResponseProcessor<T extends Output<T>> implements ResponseProcessor<T> {
 
     @Config(mandatory = true,description="The string to emit.")
+    @Deprecated
```

Comment:
Why is `name` deprecated?

Reply:
As we discussed offline, the `name` usage (where it appends something to the output) isn't really matched by functionality in any of the other `RowProcessor`s and doesn't align well with multiple output support.